src/browser/window.ts

```diff
@@ -39,7 +41,7 @@ export function buildNewWindow(app: App): void {
   // and load the index.html of the app.
   let entryBasePath = 'file://' + resolve(__dirname, '..');
   if (devMode) {
-    entryBasePath = 'http://localhost:8080';
+    entryBasePath = 'https://localhost:8080';
```

Comment:
Why do we need https here?

Reply:
Just to avoid warning security errors in the chrome console during development

Reply:
@MasterOdin I just removed this change from this PR